rocoto/sites/orion.ent

```diff
@@ -3,7 +3,7 @@
   <!ENTITY ACCOUNT         "&CPU_ACCOUNT;">
   <!ENTITY QUEUE_PE        "batch"> <!-- queue for PE (parallel environment) jobs -->
   <!ENTITY QUEUE_FORECAST  "batch"> <!-- queue for PE (parallel environment) jobs -->
-  <!ENTITY PE_EXTRA        "<native>--partition=orion</native>">
+  <!ENTITY PE_EXTRA        "<native>--partition=orion --exclusive</native>">
```

Comment:
@mrinalbiswas, do we need to make this same change on Jet and Hera, or this is only needed for Orion?

Reply:
@BinLiu-NOAA I am not sure. I think the issue was reported for Orion, we included only in orion.ent. @evankalina Anyone else reported this issue on other platforms?

Reply:
@mrinalbiswas @BinLiu-NOAA After reviewing the [RDHPCS documentation on memory usage](https://rdhpcs-common-docs.rdhpcs.noaa.gov/wiki/index.php/Running_and_Monitoring_Jobs_on_Jet_and_Hera(Theia)_-_SLURM#How_to_Get_Memory_Usage_Information) for jobs run on Jet/Hera, it is my understanding that a parallel job will automatically be given access to all of the memory on the node(s) it runs on; i.e., there is no need to specify --exclusive, except possibly for serial jobs if they need a lot of memory.

Reply:
@evankalina and @mrinalbiswas, Thanks for checking and confirming! Good to know on Jet and Hera, "--exclusive" is the default. Thanks!